project/SparkBuild.scala

```diff
@@ -254,7 +254,7 @@ object SparkBuild extends PomBuild {
   val mimaProjects = allProjects.filterNot { x =>
     Seq(
       spark, hive, hiveThriftServer, catalyst, repl, networkCommon, networkShuffle, networkYarn,
-      unsafe, testTags, sketch, mllibLocal
+      unsafe, testTags, sketch, mllibLocal, compatibility
```

Comment:
Yes. The same goes for `sketch` and `mllibLocal`. Can you file a blocker JIRA for 2.0.1 / 2.1 to make sure that we audit this post-release?